pkg/apis/mpi/v1/constants.go

```diff
@@ -17,8 +17,6 @@ package v1
 import common "github.com/kubeflow/common/pkg/apis/common/v1"
 
 const (
-	// EnvKubeflowNamespace is ENV for kubeflow namespace specified by user.
-	EnvKubeflowNamespace = "KUBEFLOW_NAMESPACE"
```

Comment:
I remember this was the default. So this isn't used anywhere anymore?

Reply:
Seems not. Let's see scripts below:

```shell
lucas@Wangs-MacBook-Pro-at-Tencent > /tmp/training-operator-master > grep -rin "KUBEFLOW_NAMESPACE" .
./docs/development/developer_guide.md:47:### Configure KUBECONFIG and KUBEFLOW_NAMESPACE
./docs/development/developer_guide.md:54:export KUBEFLOW_NAMESPACE=$(your_namespace)
./docs/development/developer_guide.md:57:- KUBEFLOW_NAMESPACE is used when deployed on Kubernetes, we use this variable to create other resources (e.g. the resource lock) internal in the same namespace. It is optional, use `default` namespace if not set.
./pkg/apis/mpi/v1/constants.go:21:	EnvKubeflowNamespace = "KUBEFLOW_NAMESPACE"
./pkg/common/constants.go:19:	EnvKubeflowNamespace = "KUBEFLOW_NAMESPACE"

 lucas@Wangs-MacBook-Pro-at-Tencent > /tmp/training-operator-master > grep -rin "EnvKubeflowNamespace" .
./pkg/apis/mpi/v1/constants.go:20:	// EnvKubeflowNamespace is ENV for kubeflow namespace specified by user.
./pkg/apis/mpi/v1/constants.go:21:	EnvKubeflowNamespace = "KUBEFLOW_NAMESPACE"
./pkg/common/constants.go:18:	// EnvKubeflowNamespace is ENV for kubeflow namespace specified by user.
./pkg/common/constants.go:19:	EnvKubeflowNamespace = "KUBEFLOW_NAMESPACE"
```



I double checked with v1.2.0 and `EnvKubeflowNamespace` is used there. It seems the transition results in a change in the `main` function, which lost this option. But we can still remove it here as it's also defined in the `kubeflow/common` library: https://github.com/kubeflow/common/blob/97658773cce10b2b706e38858e48137ae370ed10/pkg/util/util.go#L31

I shall later open another issue to restore these options.